### PR TITLE
Enable M88 android build.

### PR DESCRIPTION
**Pull request by taste1981**
This enables m88 android build.

### CODE REVIEW

File: talk/owt/patches/0015-Remove-custom-d8-dependency.patch [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+   parser.add_argument('--desugar', action='store_true')
+   parser.add_argument(
+       '--bootclasspath',
+@@ -502,8 +497,8 @@ def _CreateIntermediateDexFiles(changes, options, tmp_dir, dex_cmd):
+   if class_files:
+     # Dex necessary classes into intermediate dex files.
+     dex_cmd = dex_cmd + ['--intermediate', '--file-per-class-file']
+-    if options.desugar_dependencies:
+-      dex_cmd += ['--file-tmp-prefix', tmp_extract_dir]
++    #if options.desugar_dependencies:
```
Comment (jianjunz) [inline talk/owt/patches/0015-Remove-custom-d8-dependency.patch:33]: You can remove it if it's not needed.

File: talk/owt/patches/0015-Remove-custom-d8-dependency.patch [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+-        _rebased_errorprone_buildconfig =
+-            rebase_path("$_dep_gen_dir/$_dep_name.build_config", root_build_dir)
+-        _target_label = get_label_info(target_name, "dir")
+-        args += [
+-          "--target-name=${_target_label}",
+-          "--processorpath=@FileArg($_rebased_errorprone_buildconfig:deps_info:host_classpath)",
+-          "--enable-errorprone",
+-        ]
+-      }
++     #   _errorprone_dep = "//tools/android/errorprone_plugin:errorprone_plugin"
```
Comment (jianjunz) [inline talk/owt/patches/0015-Remove-custom-d8-dependency.patch:148]: ditto.

File: talk/owt/patches/0015-Remove-custom-d8-dependency.patch (hunk 1 above)
Comment (taste1981) [reply talk/owt/patches/0015-Remove-custom-d8-dependency.patch]: Removed

File: talk/owt/patches/0015-Remove-custom-d8-dependency.patch (hunk 2 above)
Comment (taste1981) [reply talk/owt/patches/0015-Remove-custom-d8-dependency.patch]: Removed

**Review by jianjunz — APPROVED** [review]

**Pull request merged**